scripts/analyze.sh

```diff
@@ -364,7 +364,7 @@ if [ $? -eq 0 ] || check_config_file ZJS_SENSOR; then
             echo "CONFIG_ADC=y" >> $ARCPRJFILE
             # Workaround for the Zephyr issue ZEP-1882: ADC doesn't work with
             # SDK 0.9 due to some compiler optimization, so enable debug mode.
-            if [ $SDK_VERSION = "0.9" ]; then
+            if [[ $SDK_VERSION = "0.9" ]]; then
```

Comment:
I am getting one of those unary errors without adding this, seems like it either needs to wrap $SDK_VERSION with "", or add this extra bracket on mac